### ISSUE TITLE
Cascading worldgen lag when generating new chunks

### ISSUE THREAD

**Issue by cookedpotatoes** [XCompWiz/Mystcraft-Issues#337]
Hi I'm trying to make a modpack with some friends and after exploring the world for a few minutes the game lags out and everything freezes. I checked the log and it says this:

[21:08:46] [Server thread/WARN]: Mystcraft loaded a new chunk [-47, -2] in dimension 0 (overworld) while populating chunk [-46, -1], causing cascading worldgen lag.
[21:08:46] [Server thread/WARN]: Please report this to the mod's issue tracker. This log can be disabled in the Forge config.

There are also a few other mods I have that has that error and I tried searching for how to fix this problem but it seems like there's not that much you can do to fix it. I thought I'd let you know anyways :)

**Comment by Veovis-Muaddib** [comment]
As far as I'm aware, mystcraft shouldn't be loading chunks in the overworld at all...

**Comment by cookedpotatoes** [comment]
@Veovis-Muaddib yeah I thougt that too but apparently it spawns a village house. Not sure if that is the problem though.

**Comment by Veovis-Muaddib** [comment]
Oh yes, that is true. Still, it shouldn't cause more than one or two cascades, which shouldn't cause any perceptible lag...

**Comment by cookedpotatoes** [comment]
Hmm so it's not always a problem if there's cascading worlgen lag? I tried to find all of the mods that does it and so far there's about 16 mods that does it. Some of them also spam the log with errors a lot more than others (one being Mystcraft) but I still have no idea which ones are lagging the game most. I tried using a program called VisualVM to see what was causing lag but I couldn't really find anything. When I disabled all those mods it worked fine though so I guess it's at least one of them but I don't know how to track down which one is lagging most.

**Comment by Veovis-Muaddib** [comment]
So unrelated to the cascading, if you've just started this world you may be experiencing slowdown from Mystcraft's profiling phase: http://wiki.mystcraft.xcompwiz.com/v0.13:Instability#Configuration
